src/race/execute.js

```diff
@@ -0,0 +1,24 @@
+import { RACE_MOVE } from "../constants/race.js";
+import { formatRaceState } from "./formatRaceState.js";
+import { generateRandomNumber } from "./randomNumber.js";
+
+export const executeRaces = (cars, attemptCount) => {
+  return Array.from({ length: attemptCount }, () => executeRound(cars));
+};
+
+export const executeRound = (cars) => {
+  updateCarPositions(cars);
+  return generateRaceState(cars);
+};
+
+export const updateCarPositions = (cars) => {
+  for (const carName in cars) {
```

Comment:
`cars` 배열을 순회한다면 요소 이름은 `car`를 사용하는게 조금 더 자연스러운거 같습니다!
```suggestion
  for (const car in cars) {
```
아니면 `cars`를 `carNames`로 작성해주는 것도 좋은거같아요
```suggestion
  for (const carName in carNames) {
```